README.md

```diff
@@ -18,6 +18,15 @@
 
 <hr />
 
+<div align="center">
+
+💡**Rasa Open Source 3.0 is coming up!**<br>
+[2.8](https://github.com/RasaHQ/rasa/milestone/39) will be the last minor in the 2.x series, as we need this _minor freeze_ to be able to introduce architectural changes. You can still contribute new features and improvements which will be released together with 3.0. Read more about [our contributor guidelines](#how-to-contribute).
```

Comment:
> as we need this _minor freeze_ to be able to introduce architectural changes

What does this mean? It sounds like `minor freeze` is introducing architectural changes lol

Reply:
I might add a sentence about having a number of release candidates (and maybe even alphas before then), so you will know well in advance to the 3.0 release

Reply:
I wanted to explain that the minor freeze gives us the flexibility we need to work the architectural changes into the codebase. Do you have a suggestion for formulating that? Or maybe it's not needed to explain the "why" here. I thought it'd be nice to explain to the community why this freeze happens.

for your second suggestion, sounds good. How about:
> We plan to ship alpha releases and release candidates over the next months in order to get early feedback. Stay tuned!

Reply:
> Do you have a suggestion for formulating that? 

I think it's good to include. Just not straightforward atm. Perhaps something like... "last minor in the 2.x series, as we need to pause releasing minors while we work architectural changes in 3.0."

> We plan to ship alpha releases and release candidates over the next months in order to get early feedback. Stay tuned!

Sounds good! I might adjust slightly to "next few months"

Reply:
👍🏻